src/lib/imx_ele.gni

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  chip_with_imx_ele = 0
```

Comment:
```suggestion
  chip_with_imx_ele = false
```

Reply:
Add comment of what ele is. Why is this a separate file? it seems to me like ele/trusty should probably be together in some common file. We do not generally split every configuration flat in a custom gni as that creates a lot of gni files.